### PR TITLE
show past learning circles

### PR DESCRIPTION
**Pull request by s-kennedy**
show past learning circles and fix subtitle text on courses page: https://github.com/p2pu/learning-circles/issues/703

### CODE REVIEW

**Comment by s-kennedy** [comment]
@dirkcuys this PR is dependent on https://github.com/p2pu/learning-circles/pull/702, I think that's why the tests are failing.

File: assets/react/search.jsx [hunk 1]
```diff
@@ -15,6 +15,8 @@ ReactDOM.render(
     initialState={initialState}
     searchSubject={'learningCircles'}
     Browse={BrowseLearningCircles}
+    origin={'http://localhost:8000'}
```
Comment (dirkcuys) [inline assets/react/search.jsx:18]: @s-kennedy I think this is the cause for the tests failing. The test suite doesn't make requests to the live server, it mocks the requests, but it mocks based on URL

Comment (s-kennedy) [reply assets/react/search.jsx]: @dirkcuys good catch, I removed this line but the tests are still failing.

**Comment by dirkcuys** [comment]
@s-kennedy I see the tests are still failing. Have you updated [the fixture](https://github.com/s-kennedy/p2pu-website/blob/past_lcs/test/fixtures/learning-circles.json) for the learning circles API? I suspect the search component is not receiving the data it is expecting. The easiest way is to load some example data in your local dev env, capture a request using wget/curl and use that, **but please make sure you're not checking in any personal data before you commit the fixture!**

**Comment by s-kennedy** [comment]
Oops no I hadn't updated the fixture, I'll do that when I get back.

**Comment by dirkcuys** [comment]
cool, no rush

**Pull request merged**